src/_includes/components/navigation.html

```diff
@@ -1,15 +1,15 @@
 <nav class="site-nav" aria-label="Main">
     <div class="flexer">
         <a href="{{ navigation.donate.url }}" class="c-btn c-btn--primary donate-link">{{navigation.donate.text}}</a>
-        <button class="site-nav-toggle" aria-label="Menu" id="nav-toggle" hidden>
+        <button class="site-nav-toggle" aria-label="Menu" id="nav-toggle">
```

Comment:
Removed hidden attribute to avoid layout shift in mobile view.

Before change:


https://user-images.githubusercontent.com/30730124/172020216-64752093-75b1-4367-9387-a3fb26ea06c6.mov




After change:

https://user-images.githubusercontent.com/30730124/172020131-5ef16b17-25c4-4605-bc2d-2c7d099443ac.mov